tests/components/stream/test_recorder.py

```diff
@@ -66,24 +70,25 @@ async def test_recorder_timeout(hass, hass_client):
 
         assert mock_cleanup.called
 
+        worker_sync.resume()
+        stream.stop()
+        await hass.async_block_till_done()
```

Comment:
Try doing this twice. 
```suggestion
        await hass.async_block_till_done()
        await hass.async_block_till_done()
```